androidApp/home/src/main/java/tech/takahana/iconwallpaper/android/home/ui/screen/HomeScreen.kt

```diff
@@ -50,12 +65,37 @@ fun HomeScreen() {
                     }
                 }
             )
-        }, content = { HomeConfirmScreen() }
+        },
+        content = {
+            NavHost(
+                navController = homeNavHostController,
+                startDestination = Screen.HomeSelectImageAssetContent.route
+            ) {
+                composable(Screen.HomeSelectImageAssetContent.route) {
+                    HomeSelectImageAssetContent(
+                        homeNavController = homeNavHostController,
+                        viewModel = homeSelectImageAssetViewModel
```

Comment:
上で定義しなくても `hiltViewModel()` を書くだけでも良さそうです👀

```suggestion 
                        viewModel = hiltViewModel()
```